cli/sharing.go

```diff
@@ -0,0 +1,265 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/serpent"
+)
+
+const defaultGroupDisplay = "-"
+
+func (r *RootCmd) sharing() *serpent.Command {
+	orgContext := NewOrganizationContext()
+
+	cmd := &serpent.Command{
+		Use:     "sharing [subcommand]",
+		Short:   "Commands for managing shared workspaces",
+		Aliases: []string{"share"},
+		Handler: func(inv *serpent.Invocation) error {
+			return inv.Command.HelpHandler(inv)
+		},
+		Children: []*serpent.Command{
+			r.shareWorkspace(orgContext),
+			r.showWorkspaceSharing(),
+		},
+		Hidden: true,
+	}
+
+	orgContext.AttachOptions(cmd)
+	return cmd
+}
+
+func (r *RootCmd) showWorkspaceSharing() *serpent.Command {
+	client := new(codersdk.Client)
+
+	cmd := &serpent.Command{
+		Use:     "show <workspace>",
```

Comment:
Have we considered `coder sharing status`?

Reply:
I don't mind status